### PR TITLE
Update signup footer with new text

### LINKED ISSUE #3161: Update email sign up copy
Related ticket: #2995 

After conducting some testing on usertesting.com (see related ticket) we've decided to update the email sign up button text to:

### Header
- [ ] Update button text: `Join us`
- [ ] Update body text: 
`Protect the internet as a global public resource.`
`Join our email list to take action and stay updated!`

### Footer
- [ ] Update body text:
`Protect the internet as a global public resource.`
`Join our email list to take action and stay updated!`

## A/B Test?
If possible it would nice to A/B test the button text. One version would be 'Join Us', the other would be 'Subscribe'
- [ ] Open ticket to test button language

## Desktop:
image

## Tablet:
image

## Mobile:
image

## Footer:
image

### PR DESCRIPTION
**Pull request by alanmoo**
Fixes some wrapping for default messaging too

Closes #3161, except for A/B testing of the button text. @beccaklam If you still want that, you should probably create a new ticket; It's much more to bite off.

(Test the Mozfest footer too when reviewing this, I had to make some small changes there)

### CODE REVIEW

File: source/sass/global.scss [hunk 1]
```diff
@@ -83,7 +83,7 @@ footer.site-footer {
     @media (min-width: $bp-lg) {
       column-count: 2;
       li {
-        break-inside: avoid-column;
+        break-inside: avoid;
```
Comment (alanmoo) [inline source/sass/global.scss]: This was just the wrong value for the property, and as such wasn't doing anything. It fixes a small issue in the Mozfest Footer that's currently live.

**Comment by mmmavis** [comment]
@alanmoo can you double check to make sure the bug [this PR](https://github.com/mozilla/foundation.mozilla.org/pull/3281) fixed doesn't come back?

**Comment by youriwims** [comment]
> @alanmoo can you double check to make sure the bug [this PR](https://github.com/mozilla/foundation.mozilla.org/pull/3281) fixed doesn't come back?

@mmmavis I just checked and it appears to be fixed.

**Comment by alanmoo** [comment]
> @alanmoo can you double check to make sure the bug [this PR](https://github.com/mozilla/foundation.mozilla.org/pull/3281) fixed doesn't come back?

It was still there on production, which is why I made the related change.

**Comment by alanmoo** [comment]
> Code lgtm, just out of curiosity was design updated as well? I couldn't find any references to design updates or changes in abstract/sketch.

I didn't look at a Sketch, but was on a call with Becca: I think this mainly brought the columns more in-line with the existing Sketch doc.

**Comment by alanmoo** [comment]
@beccaklam This is where Percy shines- Can you take a look at the details of the Percy failure and approve them (assuming they're appropriate)?

**Comment by beccaklam** [comment]
> @beccaklam This is where Percy shines- Can you take a look at the details of the Percy failure and approve them (assuming they're appropriate)?

Sure, can someone stay on the call after stand to show me how Percy works? @alanmoo

**Pull request merged**